### DISCUSSION TITLE
Passing arguments to App.run()

### DISCUSSION THREAD

**Discussion by jspv** [Textualize/textual#1053] [Q&A] [answered]
In the pre-CSS version, the run method supported kwargs which would be passed through to the App object.  The current (CSS) version no longer supports this.  Was there a specific reason this was removed?  Is there a more preferred way to pass arguments to the App?

**Comment by davep** [comment, marked as answer]
Have a look at https://github.com/Textualize/textual/issues/1030 -- the way to do what you're after now is, in your app, is [to override `__init__`](https://github.com/Textualize/textual/issues/1030#issuecomment-1293144656).
